src/errors.rs

```diff
@@ -86,6 +89,13 @@ impl From<SerenityError> for ParrotError {
     }
 }
 
+/// Provides an implementation to convert a [`ClientError`] to a [`ParrotError`].
```

Comment:
`"convert a rspotify [ClientError]...."`

Just for clarity because the error name is super generic. Also, we could import:

`use rspotify::ClientError as RSpotifyError` and use that as alias.